app/models/event.rb

```diff
@@ -104,6 +105,10 @@ def voice_channel_empty?
         !voice_channel.users.any?
     end
 
+    def times_have_changed?
+        !(self.start_at == self.start_at_before_last_save) || !(self.ends_at == self.ends_at_before_last_save) || !(self.date == date_before_last_save)
+    end
```

Comment:
Check also if settings have changed like channel creation; Server reminder and event notifications

Reply:
Too long, outsource in several methods

Reply:
Done